src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -224,24 +224,26 @@ export default function useBulkImportFields() {
     allAvailableFields,
     (field, idx) => allLabels.indexOf(field?.label) !== idx,
   );
-  let disambiguatedAvailableFields;
-  duplicates.forEach(duplicate => {
-    disambiguatedAvailableFields = map(
-      allAvailableFields,
-      field =>
-        field?.label === duplicate?.label
-          ? {
-              ...field,
-              label: formatDuplicateLabel(
-                field?.label,
-                field?.key,
-                intl,
-              ),
-            }
-          : field,
-      [],
-    );
-  });
+  const duplicateLabels = map(
+    duplicates,
+    duplicate => duplicate?.label,
+    [],
+  );
```

Comment:
`map` only expects two arguments, the collection to iterate over and the function invoked per iteration.
https://lodash.com/docs/4.17.15#map